australian-archaeology.csl

```diff
@@ -1,19 +1,24 @@
-<?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+﻿<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" sort-separator=" " demote-non-dropping-particle="sort-only" default-locale="en-GB" xmlns="http://purl.org/net/xbiblio/csl">
```

Comment:
```suggestion
<style class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB" xmlns="http://purl.org/net/xbiblio/csl">
```

The `sort-separator=" "` removes the comma before initials for first authors, which is incorrect.